File: Modul 3 Sem 1/C/PRAK305-2410817110005-MuhammadRizqanKarim.c
#include <stdio.h> 
int main() { 
    int detik, jam, menit, sisa; 
    printf("Masukkan jumlah detik: "); 
    scanf("%d", &detik); 
    jam = detik / 3600; 
    sisa = detik % 3600; 
    menit = sisa / 60; 
    sisa = sisa % 60; 
    if (jam >= 24) { 
    int hari = jam / 24; 
    jam = jam % 24; 
    printf("%d hari %02d:%02d:%02d\n", hari, jam, menit, sisa); 
    } else { 
    printf("%02d:%02d:%02d\n", jam, menit, sisa); 
} 
return 0; 
}

File: Modul 3 Sem 1/Phython/PRAK305-2410817110005-MuhammadRizqanKarim.py
detik = int(input("Masukkan jumlah detik: ")) 
jam = detik // 3600 
sisa = detik % 3600 
menit = sisa // 60 
sisa = sisa % 60 
if jam >= 24: 
    hari = jam // 24 
    jam = jam % 24 
    print(f"{hari} hari {jam:02}:{menit:02}:{sisa:02}") 
else: 
    print(f"{jam:02}:{menit:02}:{sisa:02}")
